### PR TITLE
fix: Added the strings of About Us fragment in different language files

### LINKED ISSUE #1996: Missing Translation of About Us page
**Describe the bug**
About us page is available only in english language.

**Expected Behaviour**

About us page describes about susi. Contents of this layout should be available in all the translation version of the app. It would help to spread why to use susi. 

**Steps to reproduce it**
Add strings values of the datas in the string files.
Use those string value to display the text.

**Screenshots of the issue**
susi_issue_aboutus

**Smartphone Info:**

|               | |
|---------------|-|
|Device         | Redmi Note 5 Pro |
|Android Version|  Oreo 8.1 |

**Would you like to work on the issue?**
Yes

### PR DESCRIPTION
**Pull request by atm1504**
Fixes #1996

Changes: 
Strings that were used in AboutUs fragment have been added in different language string files.

Screenshots for the change:

### CODE REVIEW

File: app/src/main/res/values-hi/strings.xml [hunk 1]
```diff
@@ -328,4 +329,6 @@
     <string name="placeholder_total_rating">प्लेसहोल्डर कुल रेटिंग</string>
     <string name="invalid_credentials">अवैध प्रत्यय पत्र</string>
     <string name="invalid_credentials_title">अमान्य क्रेडेंशियल्स शीर्षक</string>
+
```
Comment (batbrain7) [inline app/src/main/res/values-hi/strings.xml]: Unneeded change, remove these lines

Comment (atm1504) [reply app/src/main/res/values-hi/strings.xml]: You mean, I need to remove those 3 lines of string?

Comment (batbrain7) [reply app/src/main/res/values-hi/strings.xml]: The blank lines

Comment (atm1504) [reply app/src/main/res/values-hi/strings.xml]: Ok, I will remove those blank lines and commit soon.

**Comment by atm1504** [comment]
@batbrain7 I have removed the unwanted lines, please review it

File: app/src/main/res/values-hi/strings.xml [hunk 2]
```diff
@@ -167,7 +167,6 @@
     <string name="error_skill_listing">कौशल लाने में असमर्थ। कृपया इंटरनेट कनेक्शन की जाँच करें या बाद में पुनः प्रयास करें।</string>
     <string name="no_skill_author">कौशल लेखक नहीं मिला</string>
     <string name="reset_password_message">पासवर्ड रीसेट करना ..</string>
-
```
Comment (iamareebjamal) [inline app/src/main/res/values-hi/strings.xml]: Revert

File: app/src/main/res/values-de/strings.xml [hunk 3]
```diff
@@ -199,4 +199,21 @@
     <string name="no_device_found">Keine Geräte gefunden</string>
     <string name="setup_tut">Wenn Sie ein neues Gerät einrichten, stellen Sie sicher, dass es ist  in der Nähe und an eine Steckdose angeschlossen</string>
     <string name="no_wifi_found">Keine WiFi-Netzwerke gefunden</string>
+    <!--About Us String-->
+    <string name="url_about_susi" translatable="false">https://chat.susi.ai/overview</string>
```
Comment (iamareebjamal) [inline app/src/main/res/values-de/strings.xml]: It's same in all languages. No need to add it here

File: app/src/main/res/values-es/strings.xml [hunk 4]
```diff
@@ -151,4 +151,21 @@ Enlace de confirmación para activar su cuenta e iniciar sesión en susi.</strin
     <string name="promo_msg_template">Oye, prueba esto! - %s . Lo amarás!</string>
     <string name="error_msg_retry">Por favor, vuelva a intentarlo</string>
     <string name="logout">Logout</string>
+    <!--About Us String-->
+    <string name="url_about_susi" translatable="false">https://chat.susi.ai/overview</string>
```
Comment (iamareebjamal) [inline app/src/main/res/values-es/strings.xml]: It's clearly written translatable false

Comment (atm1504) [reply app/src/main/res/values-es/strings.xml]: Ok, I am correcting it

**Comment by atm1504** [comment]
@iamareebjamal please check it now

**Comment by atm1504** [comment]
@iamareebjamal please check it now

**Comment by atm1504** [comment]
@iamareebjamal please review it.

File: app/src/main/res/values-es/strings.xml [hunk 5]
```diff
@@ -151,4 +151,17 @@ Enlace de confirmación para activar su cuenta e iniciar sesión en susi.</strin
...
+    <string name="susi_contributors">Contribuyentess</string>
+    <string name="susi_contributors_desc">Desarrollado por <a href="https://github.com/fossasia/susi_android/graphs/contributors"><![CDATA[<u><font color='#4184f3'>Contribuyentes</u></font>]]></a>.</string>
+    <string name="susi_license_information">Licencia</string>
+    <string name="susi_license_information_desc">Este proyecto está actualmente licenciado bajo la licencia Apache versión 2.0. Según el<a href="https://github.com/fossasia/susi_android/blob/master/LICENSE"><![CDATA[<u><font color='#4184f3'>LICENSE.md</u></font>]]></a>
+    </string>
+    <string name="susi_report_issues">Informar problemas</string>
+    <string name="susi_report_issues_desc">Por favor, informe todos los problemas en<a href="https://github.com/fossasia/susi_android/issues"><![CDATA[<u><font color='#4184f3'>Github Repository Issue Tracker</u></font>]]></a>.</string>
+    <string name="susi_skill_cms">SUSI Skill CMS</string>
+    <string name="url_susi_skill_cms">https://skills.susi.ai</string>
```
Comment (iamareebjamal) [inline app/src/main/res/values-es/strings.xml]: What have you translated here?

Comment (atm1504) [reply app/src/main/res/values-es/strings.xml]: Urls cannot be translated. Moreover it was not defined in the main file that it is not translatable. So, I just copied it and pasted it here. Shall I define the links in the main string file as translatable=false?

Comment (iamareebjamal) [reply app/src/main/res/values-es/strings.xml]: If they can't be translated then you shouldn't have copied it in the first place

**Comment by atm1504** [comment]
@iamareebjamal please check it now

**Comment by iamareebjamal** [comment]
Revert unneeded changes

**Comment by atm1504** [comment]
> Revert unneeded changes

@iamareebjamal sir can you please mention what are the unneeded changes.

**Comment by iamareebjamal** [comment]
Go to file changes tab and see my reviews

**Comment by atm1504** [comment]
@iamareebjamal sir I have already reverted the unwanted changes. Please check it

**Comment by iamareebjamal** [comment]
Still not reverted

**Pull request closed without merging**